### DISCUSSION TITLE
使用style插件时，无法显示内置标签

### DISCUSSION THREAD

**Discussion by nicelike1212** [MuiseDestiny/zotero-style#1070] [Q&A]
使用style插件时，无法显示zotero7的内置标签，关闭style插件后，可以显示内置标签，我希望在使用style标签时能正常显示内置的圆形标签在标题的位置，这样可以节省屏幕标签，我是笔记本电脑，没有太多空间显示，希望能够让内置标签在题目的位置显示，希望开发者帮帮忙
![1](https://github.com/user-attachments/assets/c8c53bd7-eb47-4eea-b2e6-d02ddc578c5d)
![2](https://github.com/user-attachments/assets/d7e0c487-1be7-4b49-ab8e-73ef8b3b86f4)

**Comment by MuiseDestiny** [comment]
标题的列设置，勾选Tags
![image](https://github.com/user-attachments/assets/cbb400b2-92ed-4404-8a26-bce5cce1a133)
